spec/support/shared_examples/crud.rb

```diff
@@ -128,6 +128,21 @@
         end
       end
     end
+
+    if methods.include? :search
+      context '::search' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.